ui/src/components/SchemaForm/Widgets/TextareaWidget.tsx

```
@@ -0,0 +1,63 @@
+  disabled,
+  autofocus,
+  readonly,
+  onBlur,
+  onFocus,
+  onChange,
+  options,
+}: CustomWidgetProps<T, S, F>) => {
+  const _onChange = ({ target: { value } }: ChangeEvent<HTMLTextAreaElement>) =>
+    onChange(value === "" ? options.emptyValue : value);
+  const _onBlur = ({ target }: FocusEvent<HTMLTextAreaElement>) =>
+    onBlur(id, target && target.value);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)